examples/structured_data/advanced_techniques_for_structured_data.py

```diff
@@ -464,7 +422,7 @@ def create_deep_and_cross_model():
 """
 ## Conclusion
 
-You can use the `tf.feature_column` API with your Keras models to easily handle input categorical
+You can use the Keras Processing Layers to easily handle input categorical
```

Comment:
Suggestion:

I think you meant "preprocessing" (non-capitalized)

```suggestion
You can use the Keras prerocessing layers to easily handle input categorical
```

(Source: https://keras.io/guides/preprocessing_layers/)